src/common/Blocks/FormBlock/index.tsx

```diff
@@ -88,7 +88,7 @@ const FormBlock = ({ data }: { data: FormBlockType }) => {
     confirmationMessage?.root && convertLexicalToHTML({ ...confirmationMessage.root, format: 0 });
 
   return (
-    <div className="container mx-auto py-4 bg-white px-4">
+    <section className="container mx-auto px-4 py-8 sm:px-6 sm:py-12 lg:px-8 lg:py-16 bg-white">
```

Comment:
![image](https://github.com/OpenOceanHQ/payloadcms-website-template/assets/110245493/5e5e1a3c-35c4-40eb-b63c-af76940c0c90)

styling not proper for fields. For widths of 50%, it should be stacked side by side